File: official/docs/node/batches/manifest_batch.js
const Easypost = require('@easypost/api');

const api = new Easypost(process.env.EASYPOST_API_KEY);

api.Batch.retrieve('batch_...').then((retrievedBatch) => {
  retrievedBatch.createScanForm().then(console.log);
});

File: official/docs/go/batches/manifest_batch.go
package example

import (
    "os"

    "github.com/EasyPost/easypost-go/v2"
)

func main() {
	apiKey := os.Getenv("EASYPOST_API_KEY")
    client := easypost.New(apiKey)

    batch, err := client.CreateBatchScanForms("batch_...", "pdf")

	fmt.Println(batch) 
}